arrocke/gloss-translation
# refactor: align gloss line according to the ancient language

## What has changed

Align translation line to the left for greek and to the right for hebrew



## Connected Issues

closes #201 



## QA Steps

1. Check alignment for hebrew when the UI is in english and arabic. The translation line shouldn't change
2. Do the same for greek



## Post-Deployment
